client/src/main/kotlin/com/eightpotatoes/nexters/client/ClientApplication.kt

```diff
@@ -0,0 +1,11 @@
+package com.eightpotatoes.nexters.client
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.runApplication
+
+@SpringBootApplication(scanBasePackages = ["com.eightpotatoes.nexters"])
```

Comment:
scanBasePackages 전체로 지정한 이유가 있을까요?! 해당 모듈만 스캔하도록 수정해도 좋을 것 같은데 어떠세요?!

Reply:
@jaejae-yoo 
스크래핑한 데이터를 DB에 Insert 또는 Update해야해서 Core 모듈에 의존성이 걸리게 될 예정이라 다른 모듈과 동일하게 일괄 전체 모듈 스캔으로 작성했는데요, 조금 더 명확하게 Core와 해당 모듈 하위 패키지만 스캔하도록 변경해도 좋을 것 같습니다. 그러면 괜찮을 것 같으신가요?

```suggestion
@SpringBootApplication(scanBasePackages = ["com.eightpotatoes.nexters.client", "com.eightpotatoes.nexters.core"])
```